promscale/installation/docker.md

```diff
@@ -0,0 +1,123 @@
+# Test Promscale using a pre-built container
+If you want to try out Promscale in a test environment before getting started,
+you can use a pre-built Docker container.
+
+<highlight type="important">
+Using Promscale from a Docker container is for local testing purposes only. Do
+not use this installation method in a production environment.
```

Comment:
I'm missing a "Why/why not?" here. Without that it sounds like I should be afraid, but not of what.

Reply:
I want to add: I assume that the issue here is data retention i.e. customers may shoot themselves in the foot and lose their data. If that's the case, say so. If it's something more than that then we should also explicitly mention what it is.

Reply:
@VineethReddy02 ?

Reply:
This statement is part of our GitHub docker docs. So copy pasting it added it here. 

IMO the docker setup we suggested is two modes setting up individual containers or using the docker-compose file. Usually the day 2 operations such as scaling, upgrading are troublesome and involve multiple failure points. So assuming that's the reason for not recommending prod use cases.

Reply:
I tried to figure out where it was copy-pasted from, but I couldn't. The closest thing I could find was the warning above this section (scroll up a little): https://docs.timescale.com/timescaledb/latest/tutorials/promscale/promscale-install/#install-timescaledb.

That message and this one are not consistent.

On that page it says (reading between the lines, emphasis mine): "these _instructions_ are not necessarily suitable for a production setup", whereas here we are saying (reading between the lines, and rephrasing): "running Promscale in a Docker container is not suitable for production".

Reply:
I rewrote the page because I noticed the same thing. It should be good now. Let us know if you think it's still unclear.